src/playground/package.json

```diff
@@ -5,6 +5,6 @@
     "description": "The codebase for the ESLint PLayground (play.eslint.org)",
     "license": "MIT",
     "dependencies": {
-        "eslint": "latest"
+        "eslint": "github:eslint/eslint"
```

Comment:
This is to test Playground with an ESLint version that uses the `node:` protocol. When it's verified that the Playground works well, I'll revert this change.